sqlmesh/core/snapshot/evaluator.py

```diff
@@ -831,6 +831,10 @@ def _audit(
                 skipped=True,
             )
 
+        # Model's "blocking" argument takes precedence over the audit's default setting
+        blocking = audit_args.pop("blocking", None)
+        blocking = blocking == exp.true() if blocking else audit.blocking
```

Comment:
wonder if there's a better way to detect truthy

Reply:
Yea I was looking for a better way too, but didn't find something.. I thought of overriding `__eq__` for `Boolean` to check for `True` / `False` but didn't like the idea much. Open to suggestions.

Reply:
Does this mean we can't pass a macro result in here?

Reply:
Or I guess it will be evaluated during rendering? Can we have a test for this?

Reply:
Let me take a look, I didn't think much about this use case but it makes sense.

Reply:
Done - check out https://github.com/TobikoData/sqlmesh/pull/2813/commits/15d6cab0c01d4414d111aac80f4128966fbde3a3